x/accounts/defaults/lockup/README.md

```diff
@@ -100,10 +244,55 @@ The permanent lockup account permanently locks the coins in the account. The acc
 
 ```go
 type PermanentLockingAccount struct {
-	*BaseLockup
+	*BaseAccount
 }
 ```
 
+## How lockup work
+
+### Clawback enable account
+
+Clawback: 
+    - Determined the locked amount of token. This differ for each lockup type GetLockedCoinsWithDenoms function.
```

Comment:
```suggestion
    - Determined the locked amount of token. This different for each lockup type GetLockedCoinsWithDenoms function.
```
could you reword this? kind of confusing what is meant

Reply:
made it simpler